stablehlo/tests/interpret_tanh.mlir

```diff
@@ -1,107 +1,53 @@
-// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+// RUN: stablehlo-interpreter --interpret -split-input-file %s
 
-// CHECK-LABEL: Evaluated results of function: tanh_op_test_bf16
-func.func @tanh_op_test_bf16() -> tensor<11xbf16> {
+func.func @tanh_op_test_bf16() {
   %0 = stablehlo.constant dense<[0.0, -0.0, 1.0, 0.125, 0.1, 3.140630, 0x7F80, 0xFF80, 0x7FFF, 0x0001, 0x8001]> : tensor<11xbf16>
   %1 = stablehlo.tanh %0 : tensor<11xbf16>
-  func.return %1 : tensor<11xbf16>
-  // CHECK-NEXT: tensor<11xbf16>
-  // CHECK-NEXT: 0.000000e+00 : bf16
-  // CHECK-NEXT: -0.000000e+00 : bf16
-  // CHECK-NEXT: 7.617180e-01 : bf16
-  // CHECK-NEXT: 1.245120e-01 : bf16
-  // CHECK-NEXT: 9.960930e-02 : bf16
-  // CHECK-NEXT: 9.960930e-01 : bf16
-  // CHECK-NEXT: 1.000000e+00 : bf16
-  // CHECK-NEXT: -1.000000e+00 : bf16
-  // CHECK-NEXT: 0x7FFF : bf16
-  // CHECK-NEXT: 9.183550e-41 : bf16
-  // CHECK-NEXT: -9.183550e-41 : bf16
+  check.expect_almost_eq_const(%1, dense<[0.000000e+00, -0.000000e+00, 7.617180e-01, 1.245120e-01, 9.960930e-02, 9.960930e-01, 1.000000e+00, -1.000000e+00, 0x7FFF, 9.183550e-41, -9.183550e-41]> : tensor<11xbf16>) : tensor<11xbf16>
```

Comment:
These lines are getting very long and not super easy to read.

Reply:
Some of these checks will be removed as part of [Testing Guideline G1](https://github.com/openxla/stablehlo/blob/main/docs/reference.md#testing-guidelines )
driven by https://github.com/openxla/stablehlo/issues/1002

For now we can keep one constant per line.

Reply:
We don't have to solve this right now, but let's think how to make our tests more readable in the long term.

Reply:
I agree. We anyway need to trim most of the test cases to abide by the testing guidelines. We can work of readability of tests  as part of that.